src/fixtures/requests/application-json.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
           ],
           postData: {
             mimeType: 'application/json',
-            text: '{"number":1,"string":"f\\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}',
```

Comment:
Changed to this part of this fixture to an empty array because the empty object was throwing off some of our unit test assertions.